README.md

```diff
@@ -0,0 +1,32 @@
+## TDR Jenkinslib
+
+This is a library of groovy functions for the TDR Jenkins set up.
+
+As TDR is a multi-faceted project dealing with many Jenkins builds, a shared library of functions for Jenkins means we can limit the amount of repetitive code. It also allows devs to change code in the Jenkinslib and affect all Jenkins jobs that use that function rather than having to change functions in each Jenkins pipeline.
+
+### Jenkins projects using the Jenkinslib
```

Comment:
I'm not sure about this section. I would just list out the Jenkins jobs that will use the function if you want to keep it in. Its down to the other Jenkins jobs, if, and how they use the functions. I would be tempted not to include it at all.